app/queries/recently_updated_application_choice.rb

```diff
@@ -0,0 +1,43 @@
+# Find audits for updates to editable sections on the application form
+class RecentlyUpdatedApplicationChoice
+  def call(application_choice:, since: application_choice.created_at)
+    Audited::Audit
+      .where('audits.created_at >= :since AND audits.action = \'update\'', since: since)
+      .where(auditable_type: 'ApplicationForm', auditable_id: application_choice.application_form_id)
+      .where(application_form_audits_filter_sql)
+      .order('audits.created_at DESC')
```

Comment:
I'd assume is fine because we are filtering by the indexes:  auditable_type, auditable_id and created_at and action BUT to be cautious, I am concern with DB slow query here since this table is very large. If you run explain and/or try to run some sample queries against large data, how this class would behave in terms of performance?

Reply:
I've actually refactored it too, following some comments from @vassyz .

Reply:
Looks about right but at least once a day let's at least take a look at the DB or Skylight.

Reply:
Great work mate!